main.go

```diff
@@ -555,6 +558,23 @@ func migrateDB(ctx context.Context, sqlDB *sql.DB, dbName, dbAddress, dbUsername
 }
 
 func main() {
+	if _, err := os.Stat(".env"); !errors.Is(err, os.ErrNotExist) {
+		log.Println("Overriding ENV with .env")
+		err := godotenv.Load()
+		if err != nil {
+			log.Fatalf("Error loading .env file: %s", err)
+		}
+	}
+
+	if v, err := strconv.ParseBool(os.Getenv("PERCONA_TEST_WAIT_FOR_DEBUG_SESSION")); err == nil && v {
+		shouldWait := true // will be overridden via debug session
+		//goland:noinspection ALL
+		for shouldWait {
+			log.Println("Waiting for debugging session...")
+			time.Sleep(time.Second * 1)
+		}
+	}
```

Comment:
a way to simplify local development

here is waiting for `debug + dotEnv` in action
https://user-images.githubusercontent.com/91831719/147621927-c765975f-c645-4f84-ac8b-00676d1db517.mp4